pkg/render/dex_config.go

```diff
@@ -451,7 +451,7 @@ func (d *dexConfig) Connector() map[string]interface{} {
 		RootCASecretField: rootCASecretLocation,
 	}
 
-	if len(d.RequestedScopes()) > 0 {
+	if d.connectorType != connectorTypeOpenshift && len(d.RequestedScopes()) > 0 {
```

Comment:
I think it would be better to specify when we *do* want the scopes, so we don't make a mistake once we add more connectors, so :
`if type == google | type ==  oidc]`

Reply:
Actually, the specs other than oidc (& google) do not have the requestedscopes field, so I think you can omit the whole change.

Reply:
As we discussed this does need to be there for google as the scope does work, and I'm not sure if this will break for oidc which is why I added it this check.